lib/apress/selenium_eti/spec/company_site/eti/product_price_spec.rb

```diff
@@ -127,15 +118,8 @@
       end
 
       it 'введеная цена отображается' do
-        if @cs_eti_table.project_pulscen?
-          expect(@cs_eti_table_products.wholesale_price(@product))
-            .to eq("#{wholesale_price[:price]} руб. /шт. от #{wholesale_price[:min_qty]} шт.")
-        elsif @cs_eti_table.project_blizko?
-          expect(@cs_eti_table_products.wholesale_price(@product))
-            .to eq("#{wholesale_price[:price]} ₽ /шт. от #{wholesale_price[:min_qty]} шт.")
-        else
-          raise ArgumentError, 'locator not found'
-        end
+        expect(@cs_eti_table_products.wholesale_price(@product))
+            .to include(wholesale_price[:price].to_s, wholesale_price[:min_qty].to_s)
```

Comment:
Layout/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.